website/views/options/social/boss.jade

```diff
@@ -5,23 +5,50 @@ mixin boss(tavern, mobile)
       h3.panel-title(ng-if='group.quest.active==false')=env.t('questInvitation') + ' {{::Content.quests[group.quest.key].text()}}'
       h3.panel-title(ng-if='group.quest.active==true') {{::Content.quests[group.quest.key].text()}}
     .panel-body.modal-fixed-height
-      div(ng-if='group.quest.active==false')
-        table.table.table-striped
-          tr(ng-repeat='member in group.members')
-            td
-              span(ng-if=':: group.quest.leader && group.quest.leader==member._id && isMemberOfGroup(group.quest.leader,group) && isMemberOfPendingQuest(group.quest.leader,group)')
-                |*&nbsp;
-              |{{::member.profile.name}}
-            td {{group.quest.members[member._id] === true ? env.t('accepted') : group.quest.members[member._id] === false ? env.t('rejected') : env.t('pending')}}
+      div(ng-if='group.quest.active==false' ng-init="selectedTab = 1")
```

Comment:
Numbers were the first thing I thought of, but you're right, strings would be better. I'll change that and commit.